pkg/client/proxy.go

```diff
@@ -96,14 +96,20 @@ func NewProxyClient(ctx context.Context, ctxCancel context.CancelFunc, address s
 }
 
 const (
-	GRPCServiceTimeout = eclient.GRPCServiceTimeout * 2
+	GRPCServiceTimeout     = eclient.GRPCServiceCommonTimeout * 2
+	GRPCServiceLongTimeout = eclient.GRPCServiceLongTimeout * 2
```

Comment:
I don't think we need to doubt the long timeout. Instead, this can be `GRPCServiceLongTimeout = eclient.GRPCServiceLongTimeout + GRPCServiceTimeout`